app/helpers/forms_helper.rb

```diff
@@ -0,0 +1,18 @@
+module FormsHelper
+  def error_items(record, field)
+    return unless record.errors[field].any?
+
+    record.errors[field].map do |error|
+      { text: "#{field.to_s.humanize} #{error}" }
+    end
+  end
+
+  def error_summary_items(record)
+    record.errors.map do |error|
+      {
+        text: "#{error.attribute.to_s.humanize} #{error.message}",
+        href: "##{record.class.name.underscore}_#{error.attribute}",
```

Comment:
Is the extra `#` a typo?

Reply:
That's intentional - it's the `#` at the beginning of an HTML ID like `#recommended_link_title`